src/actuator/ActuatorFunctorsSimple.C

```diff
@@ -0,0 +1,487 @@
+  auto thrust = Kokkos::subview(actBulk_.turbineThrust_, index, Kokkos::ALL);
+  auto torque = Kokkos::subview(actBulk_.turbineTorque_, index, Kokkos::ALL);
+
+  for (int i = 0; i < 3; i++) {
+    thrust(i) = 0.0;
+    torque(i) = 0.0;
+  }
+
+  for (int j = 0; j < 3; j++) {
+    hubLoc(j) = 0.0;
+    hubOri(j) = 0.0;
+  }
+}
+
+void
+ActSimpleComputeThrustInnerLoop::operator()(
+  const uint64_t pointId,
+  const double* nodeCoords,
+  double* sourceTerm,
+  const double dual_vol,
+  const double scvIp) const
+{
+
+  auto offsets = actBulk_.turbIdOffset_.view_host();
+
+  // shouldn't thrust and torque contribs only come from blades?
+  // probably not worth worrying about since this is just a debug calculation
+
+  // determine turbine
+  /*
+  int turbId = 0;
+  const int nPointId = static_cast<int>(pointId);
+  for (; turbId < offsets.extent_int(0); turbId++) {
+    if (nPointId >= offsets(turbId)) {
+      break;
+    }
+  }
+  */
+  if (NaluEnv::self().parallel_rank()<actBulk_.num_blades_) {
+    int turbId = NaluEnv::self().parallel_rank();
+  //auto hubLoc = Kokkos::subview(actBulk_.hubLocations_, turbId, Kokkos::ALL);
+  //auto hubOri = Kokkos::subview(actBulk_.hubOrientation_, turbId, Kokkos::ALL);
+  auto thrust = Kokkos::subview(actBulk_.turbineThrust_, turbId, Kokkos::ALL);
+  auto torque = Kokkos::subview(actBulk_.turbineTorque_, turbId, Kokkos::ALL);
+
+  double r[3], rPerpShaft[3], forceTerm[3];
+
+  for (int i = 0; i < 3; i++) {
+    //forceTerm[i] = sourceTerm[i] * scvIp;
+    forceTerm[i] = sourceTerm[i]*scvIp;
+    //r[i] = nodeCoords[i] - hubLoc(i);
+    thrust(i) += forceTerm[i];
+  }
+
+  torque(0) = 0.0;
+  torque(1) = 0.0;
+  torque(2) = 0.0;
+  }
+}
+
+ActSimpleStashOrientationVectors::ActSimpleStashOrientationVectors(
+  ActuatorBulkSimple& actBulk)
+  : orientation_(helper_.get_local_view(actBulk.orientationTensor_)),
+    offset_(helper_.get_local_view(actBulk.turbIdOffset_)),
+    turbId_(actBulk.localTurbineId_)
+{
+  helper_.touch_dual_view(actBulk.orientationTensor_);
+  actBulk.turbIdOffset_.sync_host();
+}
+
+void
+ActSimpleStashOrientationVectors::operator()(int index) const
+{
+  const int pointId = index - offset_(turbId_);
+  auto localOrientation = Kokkos::subview(orientation_, index, Kokkos::ALL);
+  if(pointId>0){
+
+    // swap columns of matrix since openfast stores data
+    // as (thick, chord, span) and we want (chord, thick, span)
+    double colSwapTemp;
+    for (int i = 0; i < 9;i+=3) {
+      colSwapTemp = localOrientation(i);
+      localOrientation(i) = localOrientation(i+1);
+      localOrientation(i+1) = colSwapTemp;
+    }
+  } else {
+    // identity matrix
+    // (all other terms should have already been set to zero)
+    localOrientation(0) = 1.0;
+    localOrientation(3) = 1.0;
+    localOrientation(6) = 1.0;
+  }
+
+}
+
+void
+ActSimpleSpreadForceWhProjInnerLoop::preloop()
+{
+  actBulk_.actuatorForce_.sync_host();
+}
+
+void
+ActSimpleSpreadForceWhProjInnerLoop::operator()(
```

Comment:
You can just use the generic functor in ActuatorFunctors if you're not changing coordinate systems

Reply:
I commented out the call to the `ActSimpleSpreadForceWhProjection` functor, but left the code there in case we ever need to change coordinate systems (e.g., blades move/twist/rotate).  It can be removed very easily though, if we never use it.